persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java

```diff
@@ -346,6 +357,51 @@ public void writeTimeColumn(String columnName, String alias) {
     this.writeTimeColumnAlias = alias;
   }
 
+  public void count(String countColumnName) {
```

Comment:
The `column` methods support passing a `Column` object as a parameter. Could we allow that for `count` and standard aggregates too?

Reply:
Sure, we can. However, I don't need an API that accepts `Column` in the ticket that needed this one (https://github.com/stargate/stargate/pull/901/files) so I am not sure if we should add it.

Reply:
I guess it's a matter of convenience. In API modules I imagine we might have a `Column` object handy, so it might be worth supporting `Column` parameters if only to avoid using `Column.name()` all the time. WDYT?

Reply:
done